pkg/subctl/cmd/gather.go

```diff
@@ -171,10 +159,10 @@ func gatherConnectivity(dataType string, info gather.Info) bool {
 		gather.GlobalnetPodLogs(info)
 		gather.NetworkPluginSyncerPodLogs(info)
 	case Resources:
-		if submarinerCR != nil {
-			gather.CNIResources(info, submarinerCR.Status.NetworkPlugin)
-			gather.CableDriverResources(info, submarinerCR.Spec.CableDriver)
-			gather.OVNResources(info, submarinerCR.Status.NetworkPlugin)
+		if info.Submariner != nil {
+			gather.CNIResources(info, info.Submariner.Status.NetworkPlugin)
+			gather.CableDriverResources(info, info.Submariner.Spec.CableDriver)
+			gather.OVNResources(info, info.Submariner.Status.NetworkPlugin)
```

Comment:
```suggestion
			gather.CNIResources(info)
			gather.CableDriverResources(info)
			gather.OVNResources(info)
```

Then inside we can access info.Submariner......